bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownButton.html

```diff
@@ -4,15 +4,16 @@
     </head>
     <body>
         <wicket:panel>
-            <a class="btn dropdown-toggle" data-toggle="dropdown" wicket:id="btn" href="#">
-                <span wicket:id="icon"></span>
-                <wicket:container wicket:id="label"></wicket:container>
-            </a>
-            <ul wicket:id="dropdown-menu" class="dropdown-menu">
-                <li wicket:id="buttons">
-                    <a wicket:id="button" class="dropdown-item"></a>
-                </li>
-            </ul>
+           <a class="dropdown-toggle" wicket:id="btn" href="#" type="button" 
+		  		data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+		  		<span wicket:id="icon"></span>
+		    	<wicket:container wicket:id="label"></wicket:container>
+		  </a>
+		  <div wicket:id="dropdown-menu" class="dropdown-menu" aria-labelledby="dropdownMenuLink">
+		  	<div wicket:id="buttons">
```

Comment:
I think we should use a wicket-container otherwise it will produce an extra unwanted div:

<div wicket:id="dropdown-menu" class="dropdown-menu">
                <wicket:container wicket:id="buttons">
                    <a wicket:id="button" class="dropdown-item" href="javascript:void(0)"></a>
                </wicket:container>
            </div>

Reply:
ButtonList.populateItem(ListItem<AbstractLink> item) is overwritten with item.setRenderBodyOnly(true). So no extra unwanted div is created. 
However, using wicket:container is for sure the more elegant version. I will change to wicket:container

Reply:
That's weird because I do have the extra div in my code at the end...

Reply:
With item.setRenderBodyOnly(true) you should not. I did not have the additional div. However its now changed to wicket:container. So if you use the current DropDownButton class you will get the extra div.